node/src/effect.rs

```diff
@@ -1718,6 +1718,18 @@ impl<REv> EffectBuilder<REv> {
             .await
     }
 
+    /// Returns a list of validator status changes, by public key.
+    // TODO: Expose this via RPC.
+    pub(crate) async fn _get_consensus_validator_info(
+        self,
+    ) -> BTreeMap<PublicKey, Vec<(EraId, ValidatorChange)>>
+    where
+        REv: From<ConsensusRequest>,
+    {
+        self.make_request(ConsensusRequest::ValidatorInfo, QueueKind::Regular)
+            .await
+    }
```

Comment:
@piotr-dziubecki , a ticket needs to be created to call this method and wire the resultant data up to the json-rpc & rest status endpoints; assign to @Fraser999 to oversee and @darthsiroftardis to implement for rel 1.4